src/types/amount.ts

```diff
@@ -79,3 +77,28 @@ export class AlgoAmount {
     return new AlgoAmount({ microAlgos: amount })
   }
 }
+
+const MICROALGOS_TO_ALGOS_RATIO = 1e6
+const INVALID_MICROALGOS_ERROR_MSG = 'Microalgos should be positive and less than 2^53 - 1.'
+
+/**
+ * microalgosToAlgos converts microalgos to algos
+ * @param microalgos - number
+ * @returns number
+ */
+function microalgosToAlgos(microalgos: number) {
```

Comment:
Can we add support for bigint values?
If the type is a number, then we should check this condition, otherwise we should allow the conversion of bigints.


Worth also putting some conversion tests around AlgoAmount, given we're not dependent on SDK for this functionality now.

Reply:
There is complexity the @PatrickDinh with ensuring precision.